requirements.dev.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/xivlogs/nari.git#egg=nari
```

Comment:
This likely isn't needed since it's in the setup.py.

Reply:
Yeah this was pushed before the update to setup.py that included it.

Reply:
Removed